packages/video_player/video_player/android/src/main/java/io/flutter/plugins/videoplayer/Messages.java

```diff
@@ -1,3 +1,7 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
 // Autogenerated from Pigeon (v0.1.19), do not edit directly.
```

Comment:
Should Pigeon allow for a "prefix" for the generated file(s)? This change will probably get wiped the next time the files are auto-generated.

Reply:
If we want to ignore them, I think we could just do it by matching this line rather than naming convention. @Hixie thoughts?

Reply:
Per Discord discussion, we'll keep this (re-adding on regeneration) and revisit if that becomes problematic.